### DISCUSSION TITLE
Benifits of Github Pro (websites)

### DISCUSSION THREAD

**Discussion by OussemaNehdi** [community/community#115031] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Github free let you use github pages where the link is projectname.github.io (are they unlimited?). 
With Github Pro (i got it after i enable github student pack) i osaw a benifit saying  Get one site per GitHub account and organization, and unlimited project sites. how does that differe from the free plan? the link is it the domain name or the website can be dynamic?
i dont understand the benifits

**Comment by yoannchaudet** [comment]
The only difference with the Pro plan is that you can publish Pages sites from repositories that are private.

On the free plan, the repositories must be public.
